torch_frame/data/mapper.py

```diff
@@ -209,3 +209,59 @@ def forward(
 
     def backward(self, tensor: Tensor) -> pd.Series:
         raise NotImplementedError
+
+
+class TextTokenizationTensorMapper(TensorMapper):
+    r"""Tokenize any text series into a dictionary of
+    :class:`MultiNestedTensor`.
+
+    Args:
+        text_tokenizer (callable): A callable function that takes list of
+            strings and returns a dictionary of :class:`MultiNestedTensor`.
+            The values of the dictionary contains for example tokens and
```

Comment:
Correct me if I am wrong, but I think the set of keys should be the same for all tokenizer, i.e. ids, attention_masks and tokens. Can you be more specific in the documentation.

Reply:
No, they are not the same. For example BERT has attention mask, token input ids and sequence id. But roberta only has attention mask and token input ids. Will update the docs.